app/models/case/sar/offender.rb

```diff
@@ -233,6 +233,10 @@ def recipient_address
     (!subject_recipient?) ? postal_address : subject_address
   end
 
+  def page_count
+    DataRequest.where(case_id: self.id).joins(:data_request_logs).sum(:num_pages)
```

Comment:
Thanks for raising the data request logs issue this morning - what you did here is undoubtedly what Mo intended you to do with that data structure, but we subsequently changed it and as a result of what we found out about the behaviour today, seems this is a bug? We should probably change this to sum the `cached_num_pages` value from the data requests themselves